rskj-core/src/main/java/co/rsk/mine/gas/StableMinGasPriceProviderFactory.java

```diff
@@ -0,0 +1,43 @@
+package co.rsk.mine.gas;
+
+import co.rsk.config.mining.HttpGetStableMinGasSystemConfig;
+import co.rsk.config.mining.StableMinGasPriceSystemConfig;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+public class StableMinGasPriceProviderFactory {
+
+    private static final Logger logger = LoggerFactory.getLogger("StableMinGasPrice");
+
+    private StableMinGasPriceProviderFactory() {}
+
+    public static MinGasPriceProvider create(StableMinGasPriceSystemConfig config, DefaultMinGasPriceProvider fallBackProvider) {
```

Comment:
Do we really need to receive this default `DefaultMinGasPriceProvider`? 
If we are already receiving the configurations, we can easily initialise it with the `minGasPrice` from the config no?